gen/gen.go

```diff
@@ -181,9 +180,8 @@ func Gen(exercise string, tests map[string]interface{}, t *template.Template) er
 	formattedFileContent, err := format.Source(casesFileContent)
 	if err != nil {
 		fmt.Print("[ERROR] failed to format the output with gofmt (the generated source has a syntax error)")
-		debugFileContent := append(casesFileContent, []byte("// !NOTE: Error during source formatting: Line:Column "+fmt.Sprint(err)+"\n")...)
 		// Save the raw unformatted, error-containing source for purposes of debugging the generator.
-		_ = outputSource("ERROR", casesFile, debugFileContent)
+		_ = outputSource("ERROR", casesFile, append(casesFileContent, []byte(fmt.Sprintf("// !NOTE: Error during source formatting: Line:Column %v\n", err))...))
```

Comment:
Why was the code here changed? The old version seemed more readable. 🤔

Reply:
Linter suggestions, because `debugFileContent := append(casesFileContent, ...)` should assign the value to the same variable as used in the first parameter

Reply:
So but then why don't we use "casesFileContent" instead of "debugFileContent" but keep in on a separate line for readability?

Reply:
Changed